iOS/Podfile

```diff
@@ -19,3 +20,11 @@ target 'FlipperKit' do
   pod 'SocketRocket', '~> 0.6.0'
 
 end
+target 'PortForwardingMacApp' do
+  #use_frameworks!
+  platform :osx, '11'
+  workspace 'FlipperKit.xcworkspace'
+  project '../PortForwardingMacApp.xcodeproj'
```

Comment:
Placing the PortFrowarding project at the root level is messy. I only placed it there because the src is split between `/PortForwardingMacApp/main.m` and `/ios/FlipperKit/FKPortForwarding/*`. 

Q: should we just refactor this and place the project and relevant sources under `/desktop`?

Reply:
Decided to refactor anyway. PR now moves `/ios/FlipperKit/FKPortForwarding/FKPortForwardingClient.*` to `/desktop/PortForwardingMacApp`